src/components/InstancePlugin/InstancePlugin.test.js

```diff
@@ -0,0 +1,25 @@
+import '../../../test/jest/__mock__';
+import { screen } from '@folio/jest-config-stripes/testing-library/react';
+import InstancePlugin from './InstancePlugin';
+import { renderWithIntl } from '../../../test/jest/helpers';
+
+const onSelectMock = jest.fn();
+
+describe('InstancePlugin', () => {
+  afterEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it('should InstancePlugin', () => {
```

Comment:
What does it mean `'should InstancePlugin'`?
Maybe you meant something like `'should render InstancePlugin'`?

Reply:
Fixed